builder/build-image.sh

```diff
@@ -116,7 +116,7 @@ if tools/build.sh "$IMAGE"; then
     # All going well, upload the VMI in the registry
     # this should be done only if this is a push to main
     if test "$UPLOAD" == "true"; then
-	    builder/upload.sh "$IMAGE" "$COMMIT_SHA" "$(realpath secrets.json)"
+	    builder/upload.sh "$(realpath "$IMAGE")" "$COMMIT_SHA" "$(realpath secrets.json)"
```

Comment:
I think this won't fix the tag issue, as the problem seems that the metadata is not correctly obtained in the jq command

Reply:
mmm, ok; I thought the `jq` command failed because it depends on `MANIFEST_OUTPUT`, which in turns depends on `IMAGE`?
https://github.com/EGI-Federation/fedcloud-vmi-templates/blob/0883bf993e64951bc824792cb7837df74e7a9677/builder/upload.sh#L36-L39

Reply:
Trying to debug in #114

Reply:
I see, thanks. Reverting this commit, then, and waiting for the outcomes of #114